### ISSUE TITLE
Can't Update Molly 7.13.4-1-FOSS

### ISSUE THREAD

**Issue by benji88i** [mollyim/mollyim-android#369]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

hi there,

Thank you for making the mower secure and awesome application. I really appreciate it.

I seem to be unable to update to the latest version 

I am running 7.13.4-1- FOSS

I have tried to install updates through various means to no avail. I installed this through acrescent.

when I goto accrescent and also the app to search for updates I get none available. 

I've also tried f-droid too (even though Installed through accrescent)

looking forward to hearing from you.

sincerely 

James

### Steps to reproduce

1. Goto accrescent, choose updates. None. Clicked refresh. None available
2. Went inside Molly, went to "App updates". None available.
3. I've tried to clear cache and that doesn't work either.

### Molly version

7.13.4-1- FOSS

### Android version

android 13 - graphene OS

### Device

Google pixel 4. 

### Link to debug log

https://debuglogs.org/android/7.13.4/5b5d3088bcbe19d7d4a00abea6f2b7118c303f1836c8d4ea5393b7e2f9332920

**Comment by valldrac** [comment]
Were you able to update in the end? Just a couple of things: the version of Molly on Accrescent doesn't auto-update (to comply with Accrescent's policy). You'll need to update it through the Accrescent app.

You can also download the APK directly from GitHub and install it yourself:

https://github.com/mollyim/mollyim-android/releases

**Comment by bub0-bub0** [comment]
Hi valldrac,

I had the same problem today, but not via Accrescent and not with that old version.
I prefer GitHub and I had already tested v7.18.2-1-FOSS (runs great!).
After trying out the UnifiedPush variant of v7.17.5 yesterday, I read here on GitHub that you're integrating UnifiedPush into the main app soon (hurray, thanks!).

So I installed v7.17.5-1-FOSS again.
Could have gone straight for 7.18.2 in GitHub/Obtainium, but because of backup/restore I wanted to start with 7.17.5 and then update.

But I couldn't update to a higher version within the app (v7.17.5-1-FOSS).
If I tick both boxes (i.e. also for beta versions) and then tap on Check updates, I am still asked for the passphrase, but shortly after inserting/sending the passphrase the app crashes.

My workaround:
Updating the app Molly within Obtainium by adding as app (search via GitHub), tick the box for beta versions, regex „FOSS” for app version and go for it…
Obtainium found the latest version, asked if it should be updated and updated after confirmation.
Then I have deleted Molly from Obtainium, because I hope, the update-bug is solved and I will get next versions directly. ;-)

**Comment by bub0-bub0** [comment]
Hi valldrac,

the in-app update function doesn't work since it was added into the settings menu.
In the meantime I am using v7.28.4-1-FOSS on my Pixel phones 8 Pro and 9 Pro XL.
– There is no notification or other activity about the newer (pre) release v7.29.3-1-FOSS,
so now I have updated via Obtainium on my Pixel 8 Pro.
– On my Pixel 9 Pro XL I will wait until tomorrow, maybe it needs more time?

I am using GrapheneOS' newest release 2025010700 on my Pixel phones.

**Comment by valldrac** [comment]
The in-app updater uses the F-Droid repo to check for updates. We usually delay releasing the beta on F-Droid a bit, after publishing on GitHub. That's why it didn’t show up to you earlier. Version 7.29.3-1 is now available on F-Droid.

**Comment by bub0-bub0** [comment]
No change today either – Molly v7.28.4-1-FOSS does not find a new version and therefore does not ask me whether it should update to v7.29.3-1-FOSS. :-(

**Comment by pi99y** [comment]
I was still running 7.29.4-1-FOSS today and also couldn't update. 
If I manually pressed "check for updates" it said "Last checked: Now", but no update.

Then I enabled "beta" updates and it installed 7.44.2-1-FOSS
Are all latest updates marked as beta?

**Comment by valldrac** [comment]
> Are all latest updates marked as beta?

No, 7.44.2-1-FOSS isn't a beta version. There's just a delay between checking for updates and the install popup, that's when the download completes. So you probably still got the regular (non-beta) version.
